zerver/tornado/views.py

```diff
@@ -45,6 +49,16 @@ def notify(
 def cleanup_event_queue(
     request: HttpRequest, user_profile: UserProfile, queue_id: str = REQ()
 ) -> HttpResponse:
+    user_port = get_user_tornado_port(user_profile)
+
+    if not is_current_port(user_port):
+        assert settings.USING_RABBITMQ
+        get_queue_client().json_publish(
+            notify_tornado_queue_name(user_port),
+            {"users": [user_profile.id], "event": {"type": "cleanup_queue", "queue_id": queue_id}},
+        )
+        return json_success(request)
```

Comment:
I think this probably won't do the normal logging of the deletion event; I think it'd be correct to just move the `log_data` /  `RequestNotes` bit to add the `queue_id` above this block? 

And probably we need to make sure the `cleanup_queue` queue handler logs in a reasonable way if it gets an error, i.e. because the request had an invalid `queue_id`.

Reply:
Improved the logging.